### ISSUE TITLE
Removing xlink:href attribute node from <use> element throws error

### ISSUE THREAD

**Issue by m4w4q7** [DevExpress/testcafe-hammerhead#1901]
Removing `xlink:href` attribute with `removeAttributeNode` from a `<use>` element throws an error with Hammerhead (_"Uncaught DOMException: Failed to execute 'removeAttributeNode' on 'Element': The node provided is owned by another element."_ in Chrome and _"NotFoundError: Node was not found"_ in Firefox) but works fine without proxy.

Here's a simple example:
```html
<!DOCTYPE html>
<html>
  <head>
  </head>

  <body>

    <svg style="display: none" xmlns="http://www.w3.org/2000/svg">
      <defs>
        <g id="icon">
          <rect x="0" y="0" width="100" height="100"></rect>
        </g>
      </defs>
    </svg>

    <svg xmlns="http://www.w3.org/2000/svg">
      <use xlink:href="#icon"></use>
    </svg>

    <script>
      var use = document.querySelector('use');
      use.removeAttributeNode(use.attributes[0]);
    </script>

  </body>
</html>
```

You can check it here: https://m4w4q7.github.io/remove-attribute-node-bug-example
(an empty page without proxy and a black rectangle with console error when using proxy)

It also works if we use `href` instead of `xlink:href`, but Safari does not support it yet, so we can't change that.

This method of removing the attribute is used in recent versions of hyperHTML, and currently we can't use TestCafe with it.

**Comment by LavrovArtem** [comment]
Thank you for the detailed description. I've reproduced the problem. Our team will research it and find a suitable solution.

**Comment by JeremyCraigMartinez** [comment]
Has there been any movement on this? Or team is seeing a similar issue

**Comment by Dmitry-Ostashev** [comment]
Currently we cannot provide any estimate on this.

**Comment by carolord999** [comment]
Hi, any news about this?

**Comment by AlexSkorkin** [comment]
No news yet. Once we get any results, we will share them here.

**Comment by TrevorKarjanis** [comment]
Support for HTMLElement.removeAttributeNode has been merged and released in [testcafe-hammerhead@25.5.15](https://github.com/DevExpress/testcafe-hammerhead/releases/tag/v24.5.15). We're waiting for the root testcafe package to [uptake the version](https://github.com/DevExpress/testcafe/blob/020a5f3c7f4458fbc01808dabf77601ee44c43c2/package.json#L137) before validation can be completed.

**Comment by miherlosev** [comment]
Fixed in https://github.com/DevExpress/testcafe-hammerhead/pull/2742
